build.sbt

```
@@ -202,27 +202,33 @@ lazy val docSettings = allSettings ++ Seq(
     "gray" -> "#7D7E7D",
     "gray-light" -> "#E5E6E5",
     "gray-lighter" -> "#F4F3F4",
-    "white-color" -> "#FFFFFF"),
-  addMappingsToSiteDir(mappings in(ScalaUnidoc, packageDoc), micrositeDocumentationUrl),
+    "white-color" -> "#FFFFFF"
+  ),
+  addMappingsToSiteDir(mappings in (ScalaUnidoc, packageDoc), micrositeDocumentationUrl),
   ghpagesNoJekyll := false,
-  scalacOptions in(ScalaUnidoc, unidoc) ++= Seq(
+  scalacOptions in (ScalaUnidoc, unidoc) ++= Seq(
     "-groups",
     "-implicits",
-    "-skip-packages", "scalaz",
-    "-doc-source-url", scmInfo.value.get.browseUrl + "/tree/master€{FILE_PATH}.scala",
-    "-sourcepath", baseDirectory.in(LocalRootProject).value.getAbsolutePath,
-    "-doc-root-content", (resourceDirectory.in(Compile).value / "rootdoc.txt").getAbsolutePath
+    "-skip-packages",
+    "scalaz",
+    "-doc-source-url",
+    scmInfo.value.get.browseUrl + "/tree/master€{FILE_PATH}.scala",
```

Dollar is still the interpolation currency
```suggestion
    scmInfo.value.get.browseUrl + "/tree/master${FILE_PATH}.scala",
```